bxin/ptg2hex/hex_diagnostics.ipynb

```diff
@@ -0,0 +1,607 @@
+   "outputs": [],
+   "source": [
+    "LUTfile = '%s/notebooks/ts_config_mttcs/MTHexapod/v1/default.yaml'%(os.environ[\"HOME\"])\n",
+    "with open(LUTfile, 'r') as stream:\n",
+    "    aa = yaml.safe_load(stream)\n",
+    "if hex.salinfo.index == 1:\n",
+    "    elevCoeff = aa['camera_config']['elevation_coeffs']\n",
+    "    tCoeff = aa['camera_config']['temperature_coeffs']\n",
+    "elif hex.salinfo.index == 2:\n",
+    "    elevCoeff = aa['m2_config']['elevation_coeffs']\n",
+    "    tCoeff = aa['m2_config']['temperature_coeffs']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pred = []\n",
+    "print('Predicted LUT compensation:')\n",
+    "for i in range(6):\n",
+    "    coeff = elevCoeff[i] #starts with C0\n",
+    "    mypoly = np.poly1d(coeff[::-1]) #poly1d needs C5 first\n",
+    "    pred.append(mypoly(elev))\n",
+    "    print('%10.2f '%pred[i], end= ' ')\n",
+    "print('\\n')\n",
+    "print('Actual LUT compensation:')\n",
+    "for i in 'xyzuvw':\n",
+    "    print('%10.2f '%getattr(posC, i), end= ' ')\n",
+    "print('\\n')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Do a slew, then check the LUT again"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "location = EarthLocation.from_geodetic(lon=-70.747698*u.deg,\n",
+    "                                       lat=-30.244728*u.deg,\n",
+    "                                       height=2663.0*u.m)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "now = datetime.now()\n",
+    "print(\"Start to point the telescope\", now)\n",
+    "\n",
+    "alt = 60. * u.deg\n",
+    "az = 0. * u.deg\n",
+    "rot_tel = Angle(0, unit= u.deg) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "target_name=\"TMA motion test\"\n",
+    "time_data = await ptg.tel_timeAndDate.next(flush=True, timeout=2)\n",
+    "curr_time_ptg = Time(time_data.mjd, format=\"mjd\", scale=\"tai\")\n",
+    "time_err = curr_time_ptg - Time.now()\n",
+    "print(f\"Time error={time_err.sec:0.2f} sec\")\n",
+    "\n",
+    "print(curr_time_ptg.tai.value)\n",
+    "\n",
+    "cmd_elaz = AltAz(alt=alt, az=az, \n",
+    "                obstime=curr_time_ptg.tai, \n",
+    "                location=location)\n",
+    "cmd_radec = cmd_elaz.transform_to(ICRS)\n",
+    "# Calculating the other parameters     \n",
+    "rot_pa = rot_tel\n",
+    "#await rot.cmd_trackStart.start(timeout=30.)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#The pointing component is commanding the mount directly\n",
+    "ack = await ptg.cmd_raDecTarget.set_start(\n",
```

Comment:
Please delete `ack =`. You don't use it and it's not useful.

Reply:
I strongly suggest not saving the ack and not printing the result.

The await will raise an exception if the command fails, so the only time printing the ack.result will show anything is if the command succeeds -- thus the printed result will always be the same and is not interesting.

The fact that start returns an ack is basically historical baggage. (It is also enables some more complicated interactions that are basically never needed except perhaps in salobj unit tests).